.github/workflows/build.yml

```diff
@@ -380,6 +380,8 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ env.DEFAULT_NODE_VERSION }}
+      - name: Set up Bun
+        uses: oven-sh/setup-bun@v1
```

Comment:
why do we need bun setup in the browser unit tests?

Reply:
Fixed - excluded Bun tests from running in Browser CI